packages/app/components/Today/ProfessorCheckinButton.tsx

```diff
@@ -57,6 +65,15 @@ export default function TodayPageCheckinButton(): ReactElement {
                 {q.room}
               </ProfessorModalRadio>
             ))}
+            <ProfessorModalRadio value={-1}>
+              Other...
+              {queueToCheckInto === -1 ? (
+                <Input
+                  onChange={(e) => setQueueToBeCreated(e.target.value)}
+                  style={{ width: 100, marginLeft: 10 }}
+                />
```

Comment:
It might help to increase the width of this input box, especially if you want to name the queue something like "Professors Li's Office Hours" :)